packages/app/src/App.vue

```diff
@@ -1,17 +1,39 @@
 <template>
-  <div class="app">
-    <HelloWorld />
+  <div class="app flex">
+    <router-view />
   </div>
 </template>
 
 <script lang="ts">
-import { defineComponent } from 'vue';
-import HelloWorld from './components/HelloWorld.vue';
+import { defineComponent, onBeforeMount } from 'vue';
+import feathers from '@feathersjs/feathers';
+import socketio from '@feathersjs/socketio-client';
+import io from 'socket.io-client';
+import auth from '@feathersjs/authentication-client';
+
+const client = feathers();
```

Comment:
This is a duplicate. Have a look in `feathers.ts` we already configured feathers there.